README.md

```diff
@@ -1,24 +1,26 @@
+
 ## Required Software
 
-| ![Windows Logo](Documentation/Images/128px_Windows_logo.png)<br>[Windows SDK 18362+](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk) | ![VS Logo](Documentation/Images/128px_Visual_Studio_2019.png)<br>[Visual Studio 2019](https://visualstudio.microsoft.com/vs/) | ![CMake Logo](Documentation/Images/128px_CMake_logo.png)<br>[CMake](https://cmake.org/) | ![Unity3D logo](Documentation/Images/128px_Official_unity_logo.png)<br>[Unity 2019](https://unity.com/releases/2019-2?_ga=2.114950222.898171561.1571681098-1938809356.1563129846) | ![Python Logo](Documentation/Images/128pv_python_logo.png)<br>[Python 3+](https://www.python.org/downloads/) | ![NodeJS Logo](Documentation/Images/128px_NodeJs_Logo.png)<br>[Node.js](https://nodejs.org/en/download/) |
-| :---: | :---: | :---: | :---: | :---: | :---: |
-| Windows 10 May 2019 Update SDK to build the spatializer plugin.<br>Plugin will run on Windows 10 Fall Creator's update and higher and on HoloLens 2. | Visual Studio is used for code editing, deploying and building UWP app packages | CMake is required for generating Visual Studio 2019 projects | Unity 2019 is required to build the spatializer plugin package.<br>Plugin can be used on Unity 2018 LTS and higher versions. | Helper scripts for build and packaging use Python 3 and higher. | For UPM packaging
+| ![Windows Logo](Documentation/Images/128px_Windows_logo.png)<br>[Windows SDK 18362+](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk) | ![VS Logo](Documentation/Images/128px_Visual_Studio_2019.png)<br>[Visual Studio 2019](https://visualstudio.microsoft.com/vs/) | ![CMake Logo](Documentation/Images/128px_CMake_logo.png)<br>[CMake](https://cmake.org/) | ![Unity3D logo](Documentation/Images/128px_Official_unity_logo.png)<br>[Unity 2019](https://unity.com/releases/2019-2?_ga=2.114950222.898171561.1571681098-1938809356.1563129846) | ![Python Logo](Documentation/Images/128pv_python_logo.png)<br>[Python 3+](https://www.python.org/downloads/) | ![NodeJS Logo](Documentation/Images/128px_NodeJs_Logo.png)<br>[Node.js](https://nodejs.org/en/download/) | ![Android Logo](Documentation/Images/Android_symbol_green_RGB.png)<br>[Android NDK](https://developer.android.com/ndk/downloads) 
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| Windows 10 May 2019 Update SDK to build the spatializer plugin. | Visual Studio is used for code editing, deploying and building UWP app packages | CMake is required for generating Visual Studio 2019 projects | Unity 2019 is required to build the spatializer plugin package.<br>Plugin can be used on Unity 2018 LTS and higher versions. | Helper scripts for build and packaging use Python 3 and higher. | For UPM packaging. | Required for building Android binaries.
 
 ### Branch Guide

```

Comment:
Reference gitflow here?  https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

Reply:
Sure, I'll point it to the original blog post here - https://nvie.com/posts/a-successful-git-branching-model/